docs/source/azure/examples/functionappopencors-compute.rst

```diff
@@ -0,0 +1,20 @@
+.. _azure_examples_function_app_cors:
+
+Function Apps - Find Functions with Open CORS Policy
```

Comment:
I believe it is really all app services not just functions, probably more likely to be interested in websites in a CORs example. 

Maybe `App Services - Filter By CORS Configuration` and adjust filename.

Reply:
Good catch. I got thrown off because all of my web apps don't have CORS policies in which case the "cors" field is removed from the saved result, so I assumed that only Function Apps had the "cors" field. I added a policy to a web app and it did show up